langchain/src/agents/conversational/index.ts

```diff
@@ -0,0 +1,153 @@
+import { BaseLLM } from "../../llms/index.js";
+import { LLMChain } from "../../chains/index.js";
+import {
+  Agent,
+  Tool,
+  AgentInput,
+  StaticAgent,
+  staticImplements,
+  SerializedAgentT,
+  SerializedZeroShotAgent,
+} from "../index.js";
+import { PromptTemplate } from "../../prompts/index.js";
+import { PREFIX, SUFFIX, formatInstructions } from "./prompt.js";
+
+type SerializedFromLLMAndTools = {
+  prefix?: string;
+  suffix?: string;
+  input_variables?: string[];
+  ai_prefix?: string;
+  human_prefix?: string;
+};
+
+export type SerializedConversationalAgent = SerializedAgentT<
+  "conversational-react-description",
+  SerializedFromLLMAndTools,
+  AgentInput
+>;
+
+export type CreatePromptArgs = {
+  /** String to put before the list of tools. */
+  prefix?: string;
+  /** String to put after the list of tools. */
+  suffix?: string;
+  /** Instructions. */
+  instructions?: string;
+  /** Prefix to use for AI generated responses. */
+  aiPrefix?: string;
+  /** Prefix to use for human responses. */
+  humanPrefix?: string;
+  /** List of input variables the final prompt will expect. */
+  inputVariables?: string[];
+};
+
+type ConversationalAgentInput = AgentInput;
```

Comment:
can we do
```
export interface ConversationalAgentInput extends AgentInput {
  aiPrefix:string 
}
```
or is there a reason not to do that?

Reply:
Yeah that works nicely, thanks.